### PR TITLE
Release 1 6 3

### PR DESCRIPTION
**Pull request by Trim**
Hello,

I've made a little skeleton for a 1.6.3 release announce based on current changelog.

As I am not a native English speaker, I don't know if sentences make sense in English and which expressions exists exactly.

Finally, it's really a skeleton, because I can't make the "community" part: I don't know who to thanks for this release exactly and I don't know if you want to deliver a message to the community (as asking help, community manager, ...).

Hoping it will help the website to receive updates ;)
Adrien

### CODE REVIEW

**Comment by Alkarex** [comment]
Thanks for this first version :-) Will you have a look, @marienfressinaud ?

File: blog/Release/2017-04-25_FreshRSS_1.6.3_is_available.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Hey community!
+
+FreshRSS has recently released it's 1.6.3 update !
+
+What happend since our last blog post about 1.2.0 ?
+A lot, FreshRSS can now:
+
+* Support experimentally PostgreSQL
+* FreshRSS supports now PHP 7 and requires at least PHP 5.3.3+
```
Comment (aledeg) [inline blog/Release/2017-04-25_FreshRSS_1.6.3_is_available.md]: I think it's better to have "supports" on both lines

Comment (Alkarex) [inline blog/Release/2017-04-25_FreshRSS_1.6.3_is_available.md]: For the record: Full (non-experimental) support of PostgreSQL from FreshRSS version 1.7

**Comment by Trim** [comment]
Sorry for the late answer, it seems that Github haven't notified me about your comments and their UI don't give me feedbacks about Pull Requests I've opened...

I've just committed some modifications according to your comments. I hope you like it.
As I've written in the PR, this article is just a reword of the change logs, maybe you'll want to add some paragraph about the community activity ?

About PluXML, I don't know this engine I've just heard from @cborne that's a good way to have a blog without database.
Although, I think that's current situation with Pelican engine requires a big first step to install the blog on a local machine.
If PluXML can simplify contributions to the blog, I think it's a good idea indeed :)

Maybe that will be a good way to manage comments too, instead of using github issues.
I think git will be able to ignore comment files if you prefer to not share them on Github along side the articles.

**Comment by Trim** [comment]
I'm closing this PR.

Indeed it's already outdated and, now, release page from Github advertises users of new releases directly.

**Pull request closed without merging**